testimonials.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Typing Test</title>
+    <link rel="stylesheet" href="./assets/css/testimonials.css">
```

Comment:
Add `boxicon` css file link here so that GitHub logo in the navBar will be visible. The link is available in index.html file.